wordpress-mobile/gutenberg-mobile
# Unsupported block fallback

This PR implements a message to the gutenberg bridge when the user taps to open an unsupported block on a web view.

`Gutenberg` side PR: https://github.com/WordPress/gutenberg/pull/21150
`WPiOS` side PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/13967
`WPAndroid` side PR: https://github.com/wordpress-mobile/WordPress-Android/pull/11919

To test:

- Create a post on web with a block that is not supported on Mobile.
- Open that post on Mobile.
- Tap on the `(?)` button on the unsupported block.
- Choose the `Edit block in web browser` option.
- Check that you can edit the block in the web view that appears.
- Press `Continue` to save the changes.
- Save the changes to remote or check on Preview that the changes have been retrieved successfully.

We need to test these steps in:
- WPCom
- WPCom Atomic
- Self Hosted with Jetpack
- Self Hosted without Jetpack
  - v5.0
  - v5.1
  - v5.2
  - v5.3
  - latest

**Note:** For WP v5.0 and 5.1, the interface is not completely cleaned up. You might find extra not-needed elements. I opted to not add extra complexity to fix these cases.
